Utils/Dataflow/060_upload2virtuoso/uploadTTL.sh

```diff
@@ -133,7 +133,7 @@ upload_files () {
 upload_stream () {
   EOProcess="\0"
   
-  local delimiter=$'\n'
+  local delimiter=$'\4'
```

Comment:
As you had to specify the default value multiple times, I think it would be good to move it into some global variable to reuse here and there. Next time we have to change the default value it will save us from a mistake (forgetting to change the value here or there) and will take less time ;)